.github/workflows/release.yaml

```diff
@@ -1,15 +1,24 @@
 name: Release Gobrick
 # Invocable as a reusable workflow
 # Can be manually triggered
-on:
+on:  # yamllint disable-line rule:truthy
   workflow_call:
   workflow_dispatch:
     inputs:
       version:
         description: 'Version to release (major, minor, patch) Ex: 1.0.0'
         required: true
+  repository_dispatch:
+    types: [release-go-libs]
 jobs:
+  package-update:
+    uses: dell/common-github-actions/.github/workflows/update-libraries.yml@main
+    name: Dell Libraries Update
+    secrets: inherit
   csm-release:
+    needs: [package-update]
     uses: dell/common-github-actions/.github/workflows/csm-release-libs.yaml@main
     name: Release Go Client Libraries
+    with:
+      version: "${{ github.event.inputs.option || 'minor' }}"
```

Comment:
inputs.version

Reply:
inputs.version is old, hence updated it to new inputs.option.